packages/core/contracts/optimistic-assertor/implementation/examples/Insurance.sol

```diff
@@ -71,7 +63,13 @@ contract Insurance {
         defaultCurrency.safeTransferFrom(msg.sender, address(this), bond);
         defaultCurrency.safeApprove(address(oa), bond);
         assertionId = oa.assertTruthFor(
-            policies[policyId].insuredEvent,
+            abi.encodePacked(
+                "Insurance contract is claiming that insurance event ",
+                policies[policyId].insuredEvent,
+                " event had occurred as of ",
```

Comment:
duplicate mention of "event" as it is also before `policies[policyId].insuredEvent`